drivers/tests/TESTS/mbed_drivers/lp_ticker/main.cpp

```diff
@@ -24,21 +24,33 @@
 #error [NOT_SUPPORTED] Low power ticker not supported for this target
 #else
 
+
 using utest::v1::Case;
+using std::chrono::milliseconds;
 
-static const int test_timeout = 10;
+#define TEST_ASSERT_EQUAL_DURATION(expected, actual) \
+    do { \
+        using ct = std::common_type_t<decltype(expected), decltype(actual)>; \
+        TEST_ASSERT_EQUAL(ct(expected).count(), ct(actual).count()); \
+    } while (0)
+
+#define TEST_ASSERT_DURATION_WITHIN(delta, expected, actual) \
+    do { \
+        using ct = std::common_type_t<decltype(delta), decltype(expected), decltype(actual)>; \
+        TEST_ASSERT_INT_WITHIN(ct(delta).count(), ct(expected).count(), ct(actual).count()); \
+    } while (0)
 
 #define TICKER_COUNT 16
-#define MULTI_TICKER_TIME_MS 100
+#define MULTI_TICKER_TIME 100ms
 
 /* Due to poor accuracy of LowPowerTicker on many platforms
    there is no sense to tune tolerance value as it was in Ticker tests.
 
    Tolerance value is set to 600us for measurement inaccuracy + 5% tolerance
    for LowPowerTicker. */
-#define TOLERANCE_US(DELAY) (600 + DELAY / 20)
-
+#define TOLERANCE_US(DELAY) (600us + DELAY / 20)
 
+static const int test_timeout = 10;
```

Comment:
Is test_timeout still used anywhere? If it is should it be a Chrono type too ?

Reply:
I don't think it should be a chrono type in this case, it's used in GREENTEA_SETUP which only accepts integers.